edison/static/css/footer.css

```diff
@@ -0,0 +1,15 @@
+.page-footer {
+  background-color: #222;
+  color: #ccc;
+  padding: 60px 30px;
+}
+
+.footer-copyright {
+  color: #666;
+  padding: 60px;
+}
+
+.block {
+  display: inline-block;
+
+}
```

Comment:
Please make sure to add newlines at the end of your files, many tools enforce this for various reasons.

Most programmers' editors can automatically ensure the last character in your file is a newline character, so perhaps consider switching to a different editor then the one you're using. A good editor can save you a lot of time with automatically organizing code, providing access to documentation and refactoring tools and other things.

Also try to keep unnecessary blank lines out of the code....